pkg/providers/ovirt/mappings/merger.go

```diff
@@ -38,16 +38,62 @@ func extractMappings(externalMappingSpec *v2vv1alpha1.ResourceMappingSpec, crMap
 	return &primaryMappings, &secondaryMappings
 }
 
-func mergeMappings(primaryMappings *[]v2vv1alpha1.ResourceMappingItem, secondaryMappings *[]v2vv1alpha1.ResourceMappingItem) *[]v2vv1alpha1.ResourceMappingItem {
-	var mapping []v2vv1alpha1.ResourceMappingItem
+func mergeNetworkMappings(primaryMappings *[]v2vv1alpha1.NetworkResourceMappingItem, secondaryMappings *[]v2vv1alpha1.NetworkResourceMappingItem) *[]v2vv1alpha1.NetworkResourceMappingItem {
```

Comment:
I would prefer both `mergeNetworkMappings` and `mergeStorageMappings` to use the same common code, but without common interface or generics it's unlikely that it can be done cleanly. Definitely a good candidate for trying them when they're finally supported.;)

Reply:
Yeah, I was trying to solve it with reflection, but it was even worse IMHO, so I left the code duplication.